layout/doc/layout.md

```diff
@@ -243,3 +243,28 @@ export default {
     `,
 };
 ```
+
+## Get/set color scheme
+
+You might want to demo your own component that switches the color scheme.
+The way how the color scheme is implemented on the page depends on your Design System, therefore `dockit-layout` doesn't force you to use it's custom way and rely on the internal implementation details.
+Instead the current value of the color scheme is exposed on the component for reading and writing, so that you can initialize the current value from the documentation site value and change it when needed to let `dockit-layout` update the internals.
+
+Let's say your component is called `color-scheme-switcher`.
+Then you need to provide a way to set it's initial value and listen to changes, so the demo code might look like this:
+
+```js
+import { html } from 'lit';
+export const story = () => html`
+  <color-scheme-switcher
+    initial-color-scheme="${document.querySelector('dockit-layout')
+      .colorScheme}"
+    @color-scheme-change="${(event) => {
+      document.querySelector('dockit-layout').colorScheme =
+        event.detail.colorScheme;
+    }}"
```

Comment:
@m4dz let me know if this API suits your timings, because there might be a trap here: depends on when you read the `dockit-layout` from DOM, it might be not rendered yet.

Maybe be I'll have to export a plain function, e.g. `getInitialColorScheme(): 'light' | 'dark'` from `dockit-core/layout` for the initialisation part, so that it doesn't depend on the DOM, which is not a hard requirement for that logic. Setter will still be needed, and no reason to hide getter then too, so `getInitialColorScheme` will be just an addition.

Reply:
@m4dz hey, can you have a look? not sure if you still need it, just wondering if I should merge it or close